webots_ros2_examples/webots_ros2_examples/example_controller.py

```diff
@@ -51,6 +57,24 @@ def motor_callback(self, request, response):
         self.rightMotor.setVelocity(request.right_speed)
         return response
 
+    def cmdVel_callback(self, msg):
+        wheel_gap = 0.1  # in meter
+        wheel_radius = 0.021  # in meter
```

Comment:
To be more coherent with the other variables camelCase should be used for these variables.